wiki/plugins/links/mdx/urlize.py

```diff
@@ -95,6 +95,8 @@ def extendMarkdown(self, md, md_globals):
         md.inlinePatterns['autolink'] = UrlizePattern(URLIZE_RE, md)
 
 def makeExtension(configs=None):
+    if configs is None:
+        configs = {}
```

Comment:
@efischer19 This was an upstream change.

https://github.com/django-wiki/django-wiki/blob/master/src/wiki/plugins/links/mdx/urlize.py#L112